code/modules/mob/living/simple_mob/subtypes/vore/shadekin/shadekin.dm

```diff
@@ -36,9 +36,9 @@
 
 	say_list_type = /datum/say_list/shadekin
 
-	response_help = "pets the"
-	response_disarm = "bops the"
-	response_harm = "hits the"
+	response_help = "pets"
```

Comment:
Wasn't this refactored into a carbon mob?

Reply:
Apparently the simplemob version still exists there

Reply:
Bit redundant to keep it in, though, I think? 🤷‍♂

Reply:
The simplemob version still remains, if only for aspect of having AI-controlled kin. The carbonmobs is not a full replacement for simplemob, but an alternative.

Reply:
Fair nuff, yeah.